doc/api/tls.markdown

```diff
@@ -595,16 +600,16 @@ Creates a new client connection to the given `port` and `host` (old API) or
     CA certs of the client in PFX or PKCS12 format.
 
   - `key`: A string or `Buffer` containing the private key of the client in
-    PEM format. (Could be an array of keys).
+    PEM format. (Could be an array of keys.)
```

Comment:
While this correction is right I wonder if the whole thing couldn't be said without parentheses.